packages/web/src/_page-tests/experience/__snapshots__/typography.test.tsx.snap

```diff
@@ -3809,7 +3819,7 @@ exports[`Experience/Typography renders 1`] = `
                 }
               }
             >
-              footer.technology
+              Technology
```

Comment:
How come we switched from using variables to using plain text?  Seems more likely we would have a typo somewhere or it would lead to more work if we decided to update one of these links

Reply:
I wrapped the tests in TestProdiver to get rid of a warning about there not being an i18n context available when tests were run. 

Most test already had this these ones just were created without it.

Reply:
Beyond that benefit of using the real strings is it's closer to user behavior. Also if we had a typo in the key it will be more obvious since it will show up in the snapshots.